### PR TITLE
Added decoder for Dish Network UHF Remote 6.3

### PR DESCRIPTION
**Pull request by dtiller**
Please pull this decoder into your codebase. I also uploaded test recordings and data as well as photos in the rtl_433_tests project.

### CODE REVIEW

File: src/devices/dish_remote_6_3.c [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+static char *output_fields[] = {
+    "time",
+    "button",
+    NULL
+};
+
+r_device dish_remote_6_3 = {
+    .name          = "Dish remote 6.3 decoder",
+    .modulation    = OOK_PULSE_PPM_RAW,
+    .short_limit   = (SHORT_GAP + LONG_GAP) / 2,
```
Comment (zuckschwerdt) [inline src/devices/dish_remote_6_3.c]: Please put the values here and remove the defines

File: src/devices/dish_remote_6_3.c [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        return 0;
+    }
+
+    button = b[0] >> 2;
+    button_string = button_map[button];
+	
+    local_time_str(0, time_str);
+
+    data = data_make(
+            "time",  "", DATA_STRING, time_str,
```
Comment (zuckschwerdt) [inline src/devices/dish_remote_6_3.c]: add at least a model string, look at other devices for common output

**Comment by dtiller** [comment]
I made the requested changes and pushed the code and test data changes. 

Thanks for reviewing my pull request.

**Comment by dtiller** [comment]
Requested changes made and pushed up.

**Comment by dtiller** [comment]
Requested changes made and pushed up.

**Comment by zuckschwerdt** [comment]
Increase https://github.com/merbanan/rtl_433/blob/master/include/rtl_433.h#L46
and add the new device from `rtl_433 -h` to the README.txt.
My advice on `model` wasn't too useful, I guess. It should be a concise form of the r_device name. "Dish remote 6.3" should be ok, but dots, slashes an alike are not great with e.g. MQTT. Can't really say what the most fitting string would be. Don't use brand, it will be replaced with something like "class" someday.
Last thing: there is no message integrity (MIC), so `.disabled` should be `= 1`. Thanks!

**Comment by dtiller** [comment]
Christian,

Changes made - code and related tests pushed. 

Thanks!

PS - I'm working on another decoder, but it's driving me nuts! I can't seem to get the correct parameters. Of course 'rtl_433 -a' gives me perfect data, but I can't seem to replicate that with a -X flex or the actual driver params. I'll keep at it.

**Pull request merged**